doc/data_workflow.rst

```diff
@@ -63,7 +63,7 @@ These data are used in the `build_renewable_profiles` rule. `GEBCO <https://www.
 
 * **hydrobasins** datasets on watershed boundaries and basins, as available from HydroBASINS. These data are used to estimate the hydropower generation in the `build_renewable_profiles` rule.
 
-* **landcover** describes the shapes of world protected areas that are needed to identify in what areas no (renewable) assets can be installed. Currently are used to generate a `natura.tiff` raster. Will be deprecated once the global `natura.tiff` will be available.
+* **landcover** describes the shapes of world protected areas that are needed to identify in what areas no (renewable) assets can be installed. Were used to generate a `natura.tiff` raster. Nowadays the pre-compiled `natura.tiff` raster has global coverage, so there is no need to re-calculate it locally to being able run the modeling workflow.
```

Comment:
"Were used to generate ... " is a sentence without a word; please reformulate

Reply:
Thanks a lot! Fixed